functions/helpers.js

```diff
@@ -1,38 +1,37 @@
+const functions = require('firebase-functions');
+let SibApiV3Sdk = require('sib-api-v3-sdk');
+const sendInBlueSecretKey = functions.config().audioseva.send_in_blue.key;
+const defaultClient = SibApiV3Sdk.ApiClient.instance;
+let apiKey = defaultClient.authentications['api-key'];
+apiKey.apiKey = sendInBlueSecretKey;
 
-let checkValidity = filePath => (filePath.startsWith("mp3/") && filePath.endsWith(".mp3"))
 
-let createRefString = filePath => {
-    const parts = filePath.split('/');
-        
-    const list = parts[1];
-    const mp3 = parts[2];
-    let file_name = mp3.slice(0, -4);
 
-    return `/sqr/files/${list}/${file_name}`;
-}
+exports.checkValidMP3 = filePath => (filePath.startsWith("mp3/") && filePath.endsWith(".mp3"))
 
-exports.storeFileNameToDB = (filePath, db) => {
-    if(checkValidity(filePath)) {
-        let ref = db.ref( createRefString(filePath) );
-
-        //checking if the file already exists in the RT db
-        ref.child("status").once('value')
-        .then(snapshot => {
-            if(!snapshot.exists())
-                ref.set({status: "Spare"});
-            else
-                console.log("Existing");
-
-            return 1;
-        }).catch(err => console.log(err));
-    }
-}
 
-exports.removeFileNameFromDB = (db, dbPath) => {
+exports.removeFromDB = (db, dbPath) => {
     let ref = db.ref(dbPath);
     ref.remove()
         .then(() => console.log("Deleted."))
         .catch(error => console.log(error));
 }
 
 
+exports.sendEmail = (to, bcc, templateId, params) => {
+    let apiInstance = new SibApiV3Sdk.SMTPApi();
+    sendEmail = new SibApiV3Sdk.SendSmtpEmail();
+
+    sendEmail = {
+        to: [{ email: to }], // add a name prop if it fails to send
+        bcc,
+        templateId,
+        params
+    };
+
+    apiInstance.sendTransacEmail(sendEmail).then(function(data) {
+        console.log('API called successfully. Returned data: ');
```

Comment:
Will be too ambiguous which API was called. Be more specific.

Reply:
This was just for testing purposes, but I'll log everything just in case it will be needed later.